### PR TITLE
chore: upgrade @nestjs v7

### LINKED ISSUE #88: Remove default API version for Stripe
### Bug reports:

Not sure it's 100% a bug but I think it would be best if we just left the API version default as `null` because the Stripe library handles that by just choosing the default for you (latest version I think.) If someone wants to pin an API version they can do that themselves.

Once this is done we should pin a new version.

### PR DESCRIPTION
**Pull request by 9renpoto**
closes #88 

### All Submissions:

- [x] Have you followed the guidelines in our Contributing section?
- [x] Have you checked to ensure there aren't other open
      [Pull Requests](../../../pulls) for the same update/change?

### New Feature Submissions:

1. [x] Does your submission pass tests?
2. [x] Have you linted your code locally prior to submission?

### Changes to Core Features:

- [x] Have you added an explanation of what your changes do and why you'd like
      us to include them?
- [x] Have you written new tests for your core changes, as applicable?
- [x] Have you successfully ran tests with your changes locally?

---

_Add any extra details about your changes_

### CODE REVIEW

**Comment by 9renpoto** [comment]
@dolphub Hi, would you please review if you like :), thanks!

File: src/util/getStripeClient.ts [hunk 1]
```diff
@@ -6,19 +6,15 @@ const packageJson = require('./../../package.json');
 
 export function getStripeClient({
   apiKey,
-  apiVersion = '2019-12-03',
```
Comment (9renpoto) [inline src/util/getStripeClient.ts]: remove default versions, refs #88

**Comment by 9renpoto** [comment]
@dhaspden Hi, please review if you like :)

File: src/providers/createStripeProvider.spec.ts [hunk 2]
```diff
@@ -1,19 +1,19 @@
 import * as Stripe from 'stripe';
 
-import { stripeToken } from './../constants';
+import { stripeToken, apiVersion } from './../constants';
```
Comment (dhaspden) [inline src/providers/createStripeProvider.spec.ts:3]: Can you put `apiVersion` before `stripeToken` please?

File: src/util/getStripeClient.spec.ts [hunk 3]
```diff
@@ -1,17 +1,19 @@
 import Stripe from 'stripe';
 
 import { getStripeClient } from './getStripeClient';
+import { apiVersion } from '../constants'
```
Comment (dhaspden) [inline src/util/getStripeClient.spec.ts:4]: Can you add a semicolon and put this import before `getStripeClient`?

File: src/util/getStripeClient.spec.ts [hunk 4]
```diff
@@ -1,17 +1,19 @@
...
 
   it('should return the stripe client', () => {
-    const stripeClient = getStripeClient({ apiKey });
+    const stripeClient = getStripeClient({ apiKey, apiVersion });
     expect(stripeClient).toBeInstanceOf(Stripe);
   });
 
   it('should return the stripe client with custom options', () => {
     const stripeClient = getStripeClient({
+      apiVersion,
```
Comment (dhaspden) [inline src/util/getStripeClient.spec.ts:16]: Can you add this below `apiKey`?

File: src/util/getStripeClient.ts [hunk 5]
```diff
@@ -6,19 +6,15 @@ const packageJson = require('./../../package.json');
...
   appInfo = {
     name: packageJson.name,
     url: packageJson.repository,
     version: packageJson.version,
   },
-  typescript = true,
```
Comment (dhaspden) [inline src/util/getStripeClient.ts:15]: I'm out of the loop has this option been removed recently?

File: package.json [hunk 6]
```diff
@@ -25,20 +25,20 @@
...
     "rxjs": "^6.0.0",
-    "stripe": "^8.0.0",
-    "ts-jest": "25.1.0",
-    "typescript": "3.7.5"
+    "stripe": "^8.32.0",
+    "ts-jest": "25.2.1",
+    "typescript": "3.8.3"
   },
   "peerDependencies": {
     "@nestjs/common": "^6.0.0",
```
[the remark this answers is not in the document]
Comment (dhaspden) [reply package.json:44]: If it still works with `^6.0.0` then we can leave it!

**Comment by 9renpoto** [comment]
Fixed :)

**Comment by dhaspden** [comment]
Checked this out, looks really good! Thank you for your contribution once again 🙏.

**Pull request merged**